torntools/scripts/content/bazaar/ttBazaar.js

```diff
@@ -53,25 +53,26 @@ requireDatabase().then(() => {
 					let span = doc.new({ type: "span", class: "not-bold", text: `Could not find item. Please try using the Search function.` });
 
 					div.appendChild(span);
-					doc.find(".info-msg-cont .msg").appendChild(div);
+					requireElement(".msg.right-round").then(() => doc.find(".msg.right-round").appendChild(div), undefined);
 				}
 			}
 
 			// Max buy button
 			document.addEventListener("click", (event) => {
-				if (event.target.classList.contains("^=controlPanelButton") && event.target.getAttribute("aria-label").includes("Buy")) {
-					let parent = doc.find("[class*='buyMenu_']").parentElement;
-
-					let max_span = doc.new({ type: "span", text: "fill max", class: "tt-max-buy bold" });
-					parent.find("[class*='buy_']").parentElement.appendChild(max_span);
-
+				if (Object.values(event.target.classList).some((x) => /controlPanelButton/) && event.target.getAttribute("aria-label") && event.target.getAttribute("aria-label").indexOf("Buy") > -1) {
```

Comment:
I like the `Object.values(event.target.classList).some((x) => /controlPanelButton/)`, but it's not consistent with everything else. If the element has that class, it will also have `aria-label` already I think, so no need to check on it.